creating/creating_index.adoc

```diff
@@ -482,6 +483,73 @@ And then we need to change the CMD instruction to call _/usr/sbin/init_ to let s
+run the image.  There are also helper applications like Kubernetes that will allow you to determine the
+start up order of containers even when they are distributed.  But in the case where all the containers
+reside on a single node, systemd might just be exactly the solution.  Like with traditional services,
+system is capable of making sure services both start and in the order they are specified.  Moreover,
+any issues with startup or the container are logged like any other system service.
+
+When using systemd to manage your containers, you are really using systemd to call docker commands (and
+subsequently the docker daemon) to perform the actions.  Therefore, once you commit to using systemd
+to control a container, you will need to make sure that all stop, stop, and restart actions are
+conducted with systemd.  Failure to do so essentially decouples the docker daemon and systemd causing
+systemd to be out of sync.
+
+.Sample template for a systemd service file
+```
+[Unit]
+After=docker.service
+Requires=docker.service
+PartOf=docker.service
+After=[cite another service]
+Wants=[cite another service]
+
+[Service]
+EnvironmentFile=[path to configuration file]
+ExecStartPre=-[command to execute prior to starting]
+ExecStart=[command to execute for start]
+ExecStartPost=/usr/bin/sleep 10
+ExecStop=[command to execute for stop]
+Restart=always
+
+[Install]
+WantedBy=docker.service
+```
+
+The above configuration file is a sample service file that can be used and edited to control your image or
+container.  In the [Unit] section, you can declare other services needed by your image including the cases where
+those services are also images.
+
+In the [Service] section, you can also declare the actual commands that should be run prior to start, in the
+case of start, and in the case of stop.  These commands can either be straight base commands or docker run (or stop)
+commands as well.  Finally, if you are using a well made image that contains labels like STOP or RUN, you
+could also use the atomic command.  For example, a start command could simply be:
+
+```
+atomic run <image_name>
+```
+
+This works because the actual docker command to run that image is part of the image's metadata and atomic is
+capable of extracting it.
+
+The [Service] section also has an option for EnvironmentFile.  In a traditional, non-containerized systemd service,
+this configuration file resides in /etc/sysconfig/<service_name>.  In the case of a containerized application,
+these configuration files are not always configurable and therefore do not reside on the host's filesystem.  And
+in the case of where they are configurable, the EnvironmentFile is usually more important to how the service
+application is started.  If you are [starting the application](planning_use_systemd) within an image
+with systemd, then systemd will use /etc/sysconfig/<service_name> _within_ the image itself.
+
```

Comment:
For more information on writing unit files see link:https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/System_Administrators_Guide/sect-Managing_Services_with_systemd-Unit_Files.html[Managing Systemd unit files].